### ISSUE TITLE
Remove Linux images from fluent-bit-package

### ISSUE THREAD

**Issue by nickgerace** [rancher/rancher#33403]
[rancher/fluent-bit-package](https://github.com/rancher/fluent-bit-package) exists to fill a void in upstream fluent-bit: Windows image support. For Linux images, we should use upstream [fluent/fluent-bit](https://hub.docker.com/r/fluent/fluent-bit) or [rancher/mirrored-fluent-fluent-bit](https://hub.docker.com/r/rancher/mirrored-fluent-fluent-bit). Currently, we package unused fluent-bit images in the aforementioned repo. They should be removed as a result.

**Comment by nickgerace** [comment]
New image to be used in Logging v2: `rancher/fluent-bit:1.7.4-windows`

**Comment by nickgerace** [comment]
## QA validation

Once a Rancher RC is available, ensure that the airgapped cluster has a windows manifest, and not a Linux manifest for `rancher/fluent-bit:1.7.4`. `docker inspect` may suffice within an airgapped environment.

### Things to be aware of

Please note that the public Dockerhub repository _will_ have both a Linux and Windows image for `1.7.4`.
In the future, this will not be the case. This is only true for `1.7.4`. Future `fluent-bit` tags will be Windows-only for both airgapped _and_ public environments.

**Comment by deniseschannon** [comment]
Available to test in v2.6.0-rc1

**Comment by brendarearden** [comment]
Validation: PASSED

**Steps to reproduce:** 
1. Navigate to the assets for a previous release, in this case [v2.5.9](https://github.com/rancher/rancher/releases/tag/v2.5.9 )
2. Open the `rancher-images-digests-linux-amd64.txt`and validate the following fluent-bit images exist
```
docker.io/rancher/fluent-bit:1.6.10
docker.io/rancher/mirrored-fluent-fluent-bit:1.6.10
docker.io/rancher/mirrored-fluent-fluent-bit:1.6.10-debug
```
3. Open the `rancher-images-digests-linux-arm64.txt` and validate the following fluent-bit images exist
```
docker.io/rancher/fluent-bit:1.6.10 
docker.io/rancher/mirrored-fluent-fluent-bit:1.6.10 
docker.io/rancher/mirrored-fluent-fluent-bit:1.6.10-debug
```
4. Open the `rancher-images-digests-windows-1809.txt` and validate the following fluent-bit image exists
```
docker.io/rancher/fluent-bit:1.6.10
```
5. Open the `rancher-images-digests-windows-2004.txt` and validate the following fluent-bit image exists
```
docker.io/rancher/fluent-bit:1.6.10
```
6. Open the `rancher-images-digests-windows-20H2.txt` and validate the following fluent-bit image exists
```
docker.io/rancher/fluent-bit:1.6.10
```
7. Open the `rancher-images.txt` and validate the following fluent-bit images exist
```
rancher/fluent-bit:1.6.10
rancher/mirrored-fluent-fluent-bit:1.6.10
rancher/mirrored-fluent-fluent-bit:1.6.10-debug
```
7. Open the `rancher-windows-images.txt` and validate the following fluent-bit image exists
```
rancher/fluent-bit:1.6.10
```

**Steps to Validate:** 
1. Navigate to the assets for [v2.6.0-rc1](https://github.com/rancher/rancher/releases/tag/v2.6.0-rc1 )
2. Open the `rancher-images-digests-linux-amd64.txt`and validate that the `rancher/fluent-bit` image is not included in the linux manifest
```
docker.io/rancher/mirrored-fluent-fluent-bit:1.7.9 
docker.io/rancher/mirrored-fluent-fluent-bit:1.7.9-debug 
```
3. Open the `rancher-images-digests-linux-arm64.txt` and validate that the `rancher/fluent-bit` image is not included in the linux manifest
```
docker.io/rancher/mirrored-fluent-fluent-bit:1.7.9 
docker.io/rancher/mirrored-fluent-fluent-bit:1.7.9-debug 
```
4. Open the `rancher-images-digests-windows-1809.txt` and validate the following fluent-bit images still exists
```
docker.io/rancher/fluent-bit:1.7.4
```
5. Open the `rancher-images-digests-windows-2004.txt` and validate the following fluent-bit images still exists
```
docker.io/rancher/fluent-bit:1.7.4
```
6. Open the `rancher-images-digests-windows-20H2.txt` and validate the following fluent-bit images still exists
```
docker.io/rancher/fluent-bit:1.7.4
```
7. Open the `rancher-images.txt` and validate that the `rancher/fluent-bit` image is not included
```
rancher/mirrored-fluent-fluent-bit:1.7.9
rancher/mirrored-fluent-fluent-bit:1.7.9-debug
```
7. Open the `rancher-windows-images.txt` and validate the fluent-bit image still exists
```
rancher/fluent-bit:1.7.4
```
